sdk/finding.ts

```diff
@@ -49,8 +49,6 @@ export class Finding {
       ...this,
       severity: FindingSeverity[this.severity],
       type: FindingType[this.type],
-      labels: this.labels.map(l => Object.assign(l, {
-        entityType: EntityType[l.entityType]}))
```

Comment:
Should this have been removed?

Reply:
we should leave this as is. this is meant for printing the finding in human-readable format. this line is converting the label.entityType into a string (i believe its an integer by default)

Reply:
But I think when we create the label object, label.entityType is already converted into a string:
https://github.com/forta-network/forta-bot-sdk/blob/b9d5f577dfdb945a192fe60af4a06575115bfb49/sdk/label.ts#L45
So this line shall be duplicated, thus, when I run the test with this line not deleted, it returns the index but not the string. Oppositely, When I removed this, it returns the string correctly.

Reply:
the above line is converting a string value to an _enum value_. the Typescript type of the field `entityType` is an enum. but when we print it to console/logs, we want it to appear as a string so we convert it back

Reply:
Ahh, I think I mixed up something, just updated the branch!